style.css

```diff
@@ -1 +1,193 @@
 /*  Add your styling here */
+
+body {
+    background-color: rgb(234, 239, 241);
```

Comment:
By defining `margin: 0` on `body` you make sure that there is no unwanted spacing around your site, unless you want it to be. In that case you need to use `padding` or `margin` on the elements.